wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -1,45 +1,206 @@
--   **Each map must use at least two different combo colors.** Also, do not use the same color twice consecutively. Using the same combo colors in a row or using only one combo color makes it impossible for the player to notice and read when a combo starts or ends.
--   **Combo colors must not blend in with the map's background/storyboard/video.** This is so the approach circles are always visible to the player.
--   **There must be a new combo either on or after a group of spinners**, so that the objects on opposing sides of the spinner(s) don't form a single combo. (*osu!* checks this automatically unless you are editing `.osu` files by hand.)
+#### Rules
+
+-   **Hitobjects must never be off-screen in 4:3 aspect ratios.** Hitobjects that are even partially off-screen can create reading difficulties. Test play your map to confirm this.
+-   **Spinners must be long enough for Auto to achieve 1000 bonus score.** Short spinners are unreasonably difficult to complete.
+-   **Each map must use at least two different custom combo colors unless the default skin is forced.** The combo colors must not blend with the map's background/storyboard/video in any case. This is so hitobjects are always visible to the player and custom skin's combo colors do not blend with the background accidentally.slider being packed in too tightly. However, sliders that cross over themselves are fine as long as the borders are clearly visible.
+-   **Reverse arrows on sliders must not be completely visually obstructed by other hitobjects with the default or beatmap-specific skin.** Covering up reverse arrows on sliders can result in sliders being ambiguous to read.
+-   **You must not silence both slider ticks and slider slides together.** Low volume or blending sound samples are similarly discouraged when inaudible.
+-   **You must not use sound samples for sliderslide, sliderwhistle, and spinnerspin which do not naturally loop.** These hitsounds are continuous, meaning that their files play from start to end and loop as one continuous sound for the length of the object.
+-   **Every slider must have a clear and visible path to follow from start to end.** Sliders which overlap themselves in a way that makes any section unreadable or ambiguous cannot be used, such as burai sliders and hold sliders without straightforward slider borders. When perfectly overlapping two slider bodies, the first slider must be fully faded out before the second slider is fully faded in.
+
+#### Guidelines
+
+-   **All circles and slider heads should be snapped to distinct sounds in the music.** Adding hitobjects where there is no musical cue to justify them can result in unfitting rhythms.
+-   **Slider tick rate should be set according to the song.** For example if your song contains a section that uses ⅓ snapping only, using tick rate 2 would not be fitting for the entire map. In such cases, tick rate 1 should be used.
```

Comment:
Tickrate two puts sliderticks on 1/2 of a beat, a full 1/3 section would have off-beat sliderticks in this example so tickrate 1 should be used since both 1/2 and 1/3 land on this